src/utils.js

```diff
@@ -88,19 +88,44 @@ export function match(value, options, result) {
   return search ? search.result : null
 }
 
-export function changeDateFormat(string) {
-  let date = new Date(string)
-  let month = '' + (date.getMonth() + 1)
-  let day = '' + date.getDate()
-  let year = date.getFullYear()
-
-  if (month.length < 2) {
-    month = '0' + month
-  }
-  if (day.length < 2) {
-    day = '0' + day
+export function changeDateFormat(timestamp) {
+  let date = new Date(timestamp)
+  const formatedDate = date
+    .toString()
+    .split(' ')
+    .splice(1, 4)
+  const month = formatedDate[0].toUpperCase()
+  const day = formatedDate[1]
+  const year = formatedDate[2]
+  const time = formatedDate[3]
+  return `${month} ${day}, ${year} @ ${time}`
+}
+
+export function timeAgo(date) {
```

Comment:
In the Addresses component I've added a library for date manipulation: `date-fns`. You can use it to do this